lib/burnside/intrinsics.cc

```diff
@@ -326,21 +327,22 @@ mlir::FunctionType IntrinsicLibrary::Implementation::getFunctionType(
 
 std::string IntrinsicLibrary::Implementation::getWrapperName(Context &c) {
```

Comment:
Can this member function return a Twine?  (Maybe not since the width string rep is being consed locally.)